site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -778,6 +778,25 @@ components:
                description: The IDs of the gateway groups that this device is a member of. Note that "via" and "memberOf" must not be set at the same time. The same applies for "viaGroups" and "memberOf" which must be set at the same time too. The reason is that Eclipse Hono does not support groups of gateway groups.
             "ext":
                $ref: '#/components/schemas/Extensions'
+            "status":
+               $ref: '#/components/schemas/Status'
+
+      Status:
+         type: object
+         additionalProperties: false
+         description: Optional and written only by the device registry implementation.
+            It can not be updated by the user.
+            Any data inserted in the status section by the user in a PUT request will be ignored.
+         properties:
+            "created":
+               type: date-time
```

Comment:
I would recommend to put an explicit expectation of the format here. And put a requirement on the timezone. I would propose to always enforce UTC.